docs/pages/build/changelog.mdx

```diff
@@ -1,5 +1,76 @@
 # Changelog [What's new]
 
+## v0.2.18-alpha to `v0.4.1`, CLI v0.0.19 and more
+
+### SDK
+
+#### Breaking Changes
+- refactor: consolidate BlockEntity boolean fields into status ([#244](https://github.com/stackrlabs/sdk/issues/244) in [#247](https://github.com/stackrlabs/sdk/pull/247))
```

Comment:
These URLs will not be publicly accessible. 
Still want to keep them coz it looks cool?

Reply:
oh forgot to remove this one, updated